docs/issuer/on-chain-issuer/on-chain-tutorial.md

```diff
@@ -12,44 +12,44 @@ keywords:
   - smart contract
 ---
 
-This guide presents an example of how to use the **on-chain issuer**. We created a small application, where we communicate with Metamask to retrieve the user's balance and a claim about this balance is generated via the on-chain issuer.
+Currently, we have two approaches to issuing onhcain credentials. Using an onchain merklized issuer and using an onchain non-merklized issuer. 
 
-There are three main components in this application:
+:::note
+Currently, the non-merklized issuer is in a new iteration of development. You can find the old contacts and the old demo for non-merklized issuer here:
```

Comment:
Until we have new version merged and working on mobile we can't give links to it, otherwise we will have too much support requests coming. So just put "old" as current ones as if there were no new version coming.
And after we have everything merged and ready we can add it here.